lib/player.rb

```diff
@@ -0,0 +1,8 @@
+class Player
+  attr_accessor :name, :mark
```

Comment:
- [ ] The student uses attr_readers/attr_writers. Only uses attr_accessors if they really need it and must be aware that by using it they allow writing that property from outside the class. That variable can be set to anything.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png)) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

I notice you are accessing to the name and mark just to read the value, please, instead use a reader, it's better to use an accessor when you have to change and read the values of the variables.